__tests__/app.test.js

```diff
@@ -27,3 +37,26 @@ describe("GET /api/topics", () => {
       });
   });
 });
+
+
+describe("GET /api/articles/:article_id", () => {
+  test("200: Responds with an object containing all details of the article with the given ID", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body: { article } }) => {
+        expect(article).toEqual(
+          expect.objectContaining({
+            article_id: expect.any(Number),
+            title: expect.any(String),
+            topic: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            article_img_url: expect.any(String)
+          })
+        )
+      });
+  });
```

Comment:
This endpoint does have some error handling that we should consider:

When given an invalid article_id: 
`/api/articles/banana`

When given a valid article_id but one that does not exist:
`/api/articles/99999`

Notes on error handling:
https://l2c.northcoders.com/courses/sd-notes/back-end#sectionId=error-handling,step=intro